docs/core-concepts/test-suite.md

```diff
@@ -3,3 +3,164 @@ icon: kolena/test-suite-16
 ---
 
 # :kolena-test-suite-20: Test Case & Test Suite
```

Comment:
Anything to add to this guide? It feels like it could use more content, but I think this is an acceptable starting point.